src/pages/settings/setup.tsx

```diff
@@ -24,6 +25,8 @@ const Setup: NextPage = () => {
       <SettingsHeader />
 
       <main>
+        <ActivateAccount />
```

Comment:
Should we only render this if the Safe is undeployed?
It always triggers stuff like gasEstimation etc otherwise

Reply:
I have an early return in that component so it only renders if the safe is not deployed yet.